src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,11 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLdr: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource = {
+    fromInputStream(classLdr.getResourceAsStream(resource))(codec)
+  }
```

Comment:
if it's in conflict with the current code style - I'll remove that. But as recent discoveries prove, a missing brace could cause some .. undesired behavior ;)